pkg/get/tools.go

```diff
@@ -1139,7 +1139,7 @@ https://releases.hashicorp.com/{{.Name}}/{{.Version}}/{{.Name}}_{{.Version}}_{{$
 	{{$os = "windows"}}
 	{{- end -}}
 
-	{{.Version}}/{{.Name}}_{{.VersionNumber}}_{{$os}}_{{$arch}}.tar.gz`,
+	{{.Version}}/{{.Name}}_v{{.VersionNumber}}_{{$os}}_{{$arch}}.tar.gz`,
```

Comment:
Added this in a later commit after the url checker tests failed in the CI.  Kail seem to have changed the asset names from [v0.16.0](https://github.com/boz/kail/releases/tag/v0.16.0)